crates/node/src/chain_spec.rs

```diff
@@ -60,9 +60,12 @@ pub fn development_config(enable_manual_seal: Option<bool>, madara_path: PathBuf
         "dev",
         ChainType::Development,
         move || {
+            // Logging the development account
+            print_development_accounts(genesis_loader.clone());
+
             DevGenesisExt {
                 genesis_config: testnet_genesis(
-                    madara_path.clone(),
+                    genesis_loader.clone(),
```

Comment:
You don't think the second clone is required. Weird that clippy didn't gave you a warning

Reply:
The clone is necessary because we cannot move out of `genesis_loader`, a captured variable in an `Fn` closure.

If we try to pass a reference, we get problems inside `testnet_genesis`, because the `.into()` takes ownership, which is behind a shared reference. Then the compiler suggest a clone anyway:
![Screenshot 2023-08-29 at 18 40 04](https://github.com/keep-starknet-strange/madara/assets/48456867/bf13ec6e-e1c5-41e4-becd-3c3f5ad697cc)